crates/bevy_winit/src/lib.rs

```diff
@@ -296,3 +313,36 @@ fn handle_create_window_events(
         window_created_events.send(WindowCreated { id: window_id });
     }
 }
+
+fn handle_cursor_state_change_events(
```

Comment:
I don't think we need these events anymore. I'd prefer the Window getters/setters to be the one way to set these things